<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>struct3d</class>
 <widget class="QWidget" name="struct3d">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="Edit 3D structure" stdset="0">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>168</y>
     <width>137</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="Select type of potential for interactions between items in structure 1">Potential for 1-2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coulomb potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debye-Hückel potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dipole-dipole potential</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>19</y>
     <width>28</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lamda</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>19</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>142</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="Select type of potential for interactions between items in structure 1">Potential for structure 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coulomb potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debye-Hückel potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dipole-dipole potential</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>394</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>19</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>365</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>116</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="Select type of potential for interactions between items in structure 1">Potential for structure 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coulomb potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debye-Hückel potential</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dipole-dipole potential</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>327</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomInButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>298</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomOutButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>269</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>97</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit structure</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Structure 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Structure 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="colorButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>19</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
